README.md

```diff
@@ -22,13 +22,38 @@ Alternatively, if you don’t want the script builder GUI, you can also only dep
 
 Running a script requires a script (either as a JSON-encoded string or as an instance of `com.swisscom.aem.tools.jcrhopper.config.Script`) and a runner (`com.swisscom.aem.tools.jcrhopper.Runner`).
 
-#### Creating a script
+#### Creating a sample script
```

Comment:
I would like to keep the documentation for scripts as-is (as it’s the more common case) and add a new section that explains how to add samples.